### DISCUSSION TITLE
updater - JSON format

### DISCUSSION THREAD

**Discussion by tcastelly** [tauri-apps/tauri#2505] [General]
Hello,

Thanks to your [Github actions](https://github.com/tauri-apps/tauri-action) it's possible to make releases. These are my generated files:
- my-app.app.tgz
- my-app_0.1.0_amd64.AppImage
- my-app_0.1.0_amd64.deb
- my-app_0.1.0_x64.dmg

I noticed in the [documentation for the updater](https://tauri.studio/en/docs/usage/guides/updater/#configuration), releases have to be compressed as tar.gz.
```json
{
    "name":"v1.0.0",
    "notes":"Test version",
    "pub_date":"2020-06-22T19:25:57Z",
    "platforms": {
        "darwin": {
            "signature":"",
            "url":"https://github.com/lemarier/tauri-test/releases/download/v1.0.0/app.app.tar.gz"
        },
        "linux": {
            "signature":"",
            "url":"https://github.com/lemarier/tauri-test/releases/download/v1.0.0/app.AppImage.tar.gz"
        },
        "win64": {
            "signature":"",
            "url":"https://github.com/lemarier/tauri-test/releases/download/v1.0.0/app.x64.msi.zip"
        }
    }
}
```
I'm wondering if it's mandatory to compress them.

Thank you very much.

**Comment by nothingismagick** [comment]
Yes, Github won't accept binary type uploads directly.

**Reply by tcastelly** [reply]
Thank you for your answer. 
In which use case it's not possible to upload binary directly? Because It's possible to have binary files inside a release. And it's already the case with the Tauri Github Actions.

I just thought it can be easier to be able to use the Tauri Updater with the releases of the Tauri Github Actions :)
